concepticondata/conceptlists/Nagano-2013-1256.tsv

```diff
@@ -0,0 +1,1257 @@
+Nagano-2013-1256-2	2	0004	have a headache (v)			
+Nagano-2013-1256-3	3	0005	stubborn/People say he does not listen.			
+Nagano-2013-1256-4	4	0007	hair of the head	2678	HAIR OR FUR	
+Nagano-2013-1256-5	5	0009	brain	1583	BRAIN	
+Nagano-2013-1256-6	6	0011	forehead	123	FOREHEAD	
+Nagano-2013-1256-7	7	0013	eye	1248	EYE	
+Nagano-2013-1256-8	8	0014	eyeball			
+Nagano-2013-1256-9	9	0016	eyebrow	1173	EYEBROW	
+Nagano-2013-1256-10	10	0017	eyelash	1540	EYELASH	
+Nagano-2013-1256-11	11	0019	tears	1651	TEAR (OF EYE)	
+Nagano-2013-1256-12	12	0021	blind	1653	BLIND	
+Nagano-2013-1256-13	13	0022	blind	1653	BLIND	
+Nagano-2013-1256-14	14	0024	nose	1221	NOSE	
+Nagano-2013-1256-15	15	0025	snivel	208	NASAL MUCUS (SNOT)	
+Nagano-2013-1256-16	16	0027	ear	1247	EAR	
+Nagano-2013-1256-17	17	0028	ear wax	32	EARWAX	
+Nagano-2013-1256-18	18	0030	deaf	996	DEAF	
+Nagano-2013-1256-19	19	0032	mouth	674	MOUTH	
+Nagano-2013-1256-20	20	0034	lips	478	LIP	
+Nagano-2013-1256-21	21	0036	tongue	1205	TONGUE	
+Nagano-2013-1256-22	22	0038	dumb	1492	DUMB	
+Nagano-2013-1256-23	23	0040	tooth	1380	TOOTH	
+Nagano-2013-1256-24	24	0041	decay (v)			
+Nagano-2013-1256-25	25	0043	spittle	1503	SALIVA	
+Nagano-2013-1256-26	26	0044	saliva	1503	SALIVA	
+Nagano-2013-1256-27	27	0045	phlegm	3061	PHLEGM	
+Nagano-2013-1256-28	28	0047	breathe (v)	1407	BREATHE	
+Nagano-2013-1256-29	29	0049	voice	1363	VOICE	
+Nagano-2013-1256-30	30	0051	cough			
+Nagano-2013-1256-31	31	0053	cough (v)	879	COUGH	
+Nagano-2013-1256-32	32	0055	sneeze			
+Nagano-2013-1256-33	33	0057	sneeze (v)	1621	SNEEZE	
+Nagano-2013-1256-34	34	0059	yawn	1028	YAWN	
+Nagano-2013-1256-35	35	0060	hiccup (v)	162	HICCOUGH	
+Nagano-2013-1256-36	36	0061	belch (v)	2245	BELCH	
+Nagano-2013-1256-37	37	0063	jaw, chin	1510	CHIN	
+Nagano-2013-1256-38	38	0065	face	1560	FACE	
+Nagano-2013-1256-39	39	0066	feel shamed (v)	1894	ASHAMED	
+Nagano-2013-1256-40	40	0068	cheek	1730	CHEEK	
+Nagano-2013-1256-41	41	0069	temple	1604	TEMPLES	
+Nagano-2013-1256-42	42	0071	mustache	1165	MOUSTACHE	
+Nagano-2013-1256-43	43	0072	beard	678	BEARD	
+Nagano-2013-1256-44	44	0073	whiskers			腮须  sideburns
+Nagano-2013-1256-45	45	0075	neck	1333	NECK	
+Nagano-2013-1256-46	46	0076	neck	1333	NECK	
+Nagano-2013-1256-47	47	0077	nape	1347	NAPE (OF NECK)	
+Nagano-2013-1256-48	48	0079	throat	1346	THROAT	
+Nagano-2013-1256-49	49	0080	throat	1346	THROAT	
+Nagano-2013-1256-50	50	0082	shoulder	1482	SHOULDER	
+Nagano-2013-1256-51	51	0083	shoulder blade	172	SHOULDERBLADE	
+Nagano-2013-1256-52	52	0085	arm	1673	ARM	
+Nagano-2013-1256-53	53	0087	elbow	981	ELBOW	
+Nagano-2013-1256-54	54	0089	wrist	799	WRIST	
+Nagano-2013-1256-55	55	0091	hand	1277	HAND	
+Nagano-2013-1256-56	56	0093	palm	1183	PALM OF HAND	
+Nagano-2013-1256-57	57	0095	finger	1303	FINGER	
+Nagano-2013-1256-58	58	0101	nail	1258	FINGERNAIL	
+Nagano-2013-1256-59	59	0103	fist	1473	FIST	
+Nagano-2013-1256-60	60	0105	breast	1592	CHEST	
+Nagano-2013-1256-61	61	0106	breast (of woman)	1402	BREAST	
+Nagano-2013-1256-62	62	0107	mother's milk			
```

Comment:
2129 BREAST OR MILK would fit going for its description, for the gloss not so much.

Reply:
But they don't colexify in the reported languages, as far as I can tell: in fact, it was probably added to the entries because in most cases "(animal) milk" and "(human) milk" don't colexify.

Reply:
Gotcha